pkg/operator/starter.go

```diff
@@ -638,12 +638,13 @@ func prepareOauthAPIServerOperator(ctx context.Context, controllerContext *contr
 		v1helpers.CachedSecretGetter(operatorCtx.kubeClient.CoreV1(), operatorCtx.kubeInformersForNamespaces),
 	).WithAPIServiceController(
 		"openshift-apiserver",
+		"openshift-oauth-apiserver",
 		func() ([]*apiregistrationv1.APIService, []*apiregistrationv1.APIService, error) {
 			return apiServices(), nil, nil
 		},
 		apiregistrationInformers,
 		apiregistrationv1Client.ApiregistrationV1(),
-		operatorCtx.kubeInformersForNamespaces.InformersFor("openshift-oauth-apiserver"),
+		operatorCtx.kubeInformersForNamespaces,
```

Comment:
This seems wrong. Why do you change it?

Reply:
I changed the syntax of this method in https://github.com/openshift/library-go/pull/1694